fluidefi/fluidefi-caspernet-aggregator-tools
# Postgres as a container

Summary of change:
- Added docker compose file to build postgres container for local development.

Steps for testing:
- First make sure your local git is up to date
```bash
git pull
```
- Checkout the branch `feature/postgres-as-a-container`
```bash
git checkout feature/postgres-as-a-container
```
- Build the container
```bash
docker-compose up --build
```

- After the container is built, make sure you can access the docker container by running the following:

```bash
docker exec -it <container-id> /bin/sh
```

[for more information ](https://docs.docker.com/engine/reference/commandline/exec/)
